src/components/bcros/header/Index.vue

```diff
@@ -4,10 +4,7 @@
     class="bg-bcGovColor-darkBlue border-b-2 border-yellow-400"
     data-cy="bcros-header"
   >
-    <div
-      id="bcros-main-header__container"
-      class="flex flex-wrap content-center h-[68px] app-inner-container"
-    >
+    <div id="bcros-main-header__container">
```

Comment:
Is there a reason for removing these classes? It seems the header lost some vertical padding.